packages/tuono/src/ssr/server.tsx

```diff
@@ -47,49 +40,21 @@ export function serverSideRendering(routeTree: RouteTree) {
     >
 
     const mode = serverProps.mode as Mode
-    const jsBundles = serverProps.jsBundles as Array<string>
-    const cssBundles = serverProps.cssBundles as Array<string>
```

Comment:
Since those two properties are not needed, 
I think we can remove the Rust code that is creating them

Reply:
They are needed!!!

During dev they are just ignored. On prod they Server side render the bundles of the requested page.

> My branch wasn't ready at all...

Reply:
Added a new task in the PR